models/models.py

```diff
@@ -38,18 +40,22 @@ class Question(BaseModel):
     options: list[QuestionOption] = []
     is_optional: bool = False
 
-    def get_answer_text(self) -> str:
-        if self.type == QuestionType.MULTIPLE_CHOICE or self.type == QuestionType.SINGLE_CHOICE:
-            return "\n".join(opt.text for opt in self.options if opt.is_selected)
-        if self.type == QuestionType.TEXTBOX:
-            return self.textbox.value
-        return ""
+
+class Answer(BaseModel):
+    question_name: str
+    value: QuestionTextbox | List[QuestionOption]
+
+
+class FilledPoll(BaseModel):
+    answers: list[Answer]
+    user_about: User
+    user_filling: User
 
 
 class Poll(BaseModel):
     title: str
     description: str
-    questions: list[Question]
+    questions: Dict[str, Question]
```

Comment:
Konsultowałeś te zmiany modelu z frontowcami?

Reply:
To ja w sumie w imieniu frontowców się zapytam, jak to będzie reprezentowane - po czym będą mapowane te pytania?

Reply:
Ja to widzę jako słownik, gdzie kluczem jest nazwa pytania, a wartością owo pytanie. Nie wiem na ile spodziewamy się pytań, których nie trzeba uzupełniać. Wówczas, przy braku odpowiedzi słownik będzie chyba lepszy niż lista, żeby ustalić które pytanie i która odpowiedź jest do czego.

Reply:
Kolejki są teraz w innym miejscu. Także ten słownik może zostać jak dla mnie